js/README.md

```diff
@@ -19,20 +19,49 @@ are used for code-formatting and linting features for developers:
 - .eslintrc.js
 - .clang-format
 
-#### Using VSCode:
+Please follow the steps described below to setup development environment.
 
-1. in `<ORT_ROOT>/js`, run:
-   > npm ci
-2. use VSCode to open folder `<ORT_ROOT>/js`
-3. install VSCode extension if not installed yet:
-   - Clang-Format
-   - ESLint
+### Prerequisites
 
-To populate typescript type declarations, in each projects, run `npm ci`.
+- Node.js (14.0+): https://nodejs.org/ - (Optional) Use nvm ([Windows](https://github.com/coreybutler/nvm-windows) / [Mac/Linux](https://github.com/creationix/nvm)) to install Node.js
```

Comment:
make https://nodejs.org as a link

Reply:
this will be interpreted as link, so as the ones for python and vscode.

see preview: https://github.com/microsoft/onnxruntime/pull/7852/files/b84271d399b8859ec8e797018cc532a6cd9f7639?short_path=3091ee1#diff-3091ee119dad49ff3e38ad9b8e62d32319f6d898b6e09d48d6a4c3efcae4d0b9